src/lib/utils/index.js

```diff
@@ -0,0 +1,11 @@
+// Disabling because we intend to have more exports in the future.
+/* eslint-disable import/prefer-default-export */
+
+/**
+ * Checks whether a method exists in a given object
+ *
+ * @param  {String} methodName   The name of the method being checked.
+ * @param  {Object} object       The object that the method should exist in.
```

Comment:
```suggestion
 * @param  {Object} object       The object that the method should exist in
```

Reply:
This ends in a preposition and I'm shocked you didn't catch it.

Reply:
That is one rule up with which I shall not put.